cmd/swarmctl/cluster/update.go

```diff
@@ -114,6 +119,11 @@ var (
 				if err != nil {
 					return err
 				}
+
+				if hbPeriod < time.Duration(0) {
```

Comment:
I think it is improper to check in the backend in this place. Here is my reason: although Duration is negative, swarmctl will convert it into a **unit64** which is a unsigned number. As a result, check at backend will not work.